manager/integration/tests/common.py

```diff
@@ -6111,3 +6111,13 @@ def wait_for_instance_manager_count(client, number, retry_counts=120):
         time.sleep(RETRY_INTERVAL_LONG)
 
     return len(ims)
+
+
+def wait_delete_dm_device(api, name):
+    for i in range(RETRY_COUNTS):
+        path = os.path.join("/dev/mapper" + name)
+        found = os.path.exists(path)
+        if not found:
+            break
+        time.sleep(RETRY_INTERVAL)
+    assert not found
```

Comment:
Suggest include an error message for better error reporting.

Reply:
Could you elaborate more on or add an example for the comment? The implementation follows. the convention of similar functions in common.py. The traceback log message might be enough for debugging.

Reply:
cc @c3y1huang

Reply:
`assert not found` doesn't output what is not found when it fails, so suggest adding a message, for example:
```
assert not found, f"Failed to delete {path}"
```